libraries/lib-audio-io/AudioIO.cpp

```diff
@@ -2348,14 +2348,18 @@ bool AudioIO::AILAIsActive() {
 }
 
 void AudioIO::AILASetStartTime() {
-   mAILAAbsolutStartTime = Pa_GetStreamTime(mPortStreamV19);
-   wxPrintf("START TIME %f\n\n", mAILAAbsolutStartTime);
+   mAILAAbsoluteStartTime = AudioIO::Get()->GetClockTime();
+   wxPrintf("START TIME %f\n\n", mAILAAbsoluteStartTime);
 }
 
 double AudioIO::AILAGetLastDecisionTime() {
    return mAILAAnalysisEndTime;
 }
 
+double AudioIO::GetClockTime() const {
```

Comment:
`GetStreamTime` returns stream time specific to Audacity, `GetClockTime` - actual stream time. Quite confusing, and I don't understand why is it called "clock" time.